pkg/webui/components/key-value-map/entry.js

```diff
@@ -84,6 +84,7 @@ class Entry extends React.Component {
     return (
       <div className={style.entriesRow}>
         <Input
+          data-test-id={'key'}
```

Comment:
Why not relying on the `name` attribute which is unique? With this approach each entry will have the same `data-test-id` attribute.